Footer/src/footer-dark-05.html

```diff
@@ -0,0 +1,78 @@
+<div style="margin: 20px;">
```

Comment:
remove this `style` tag and use tailwind classes instead I guess other than that everything looks fine.